requirements.txt

```diff
@@ -4,3 +4,4 @@ dataclasses
 mako
 odoorpc
 openupgradelib
+sentry_sdk<=1.9.0
```

Comment:
At the moment 2.20 is the latest version.
https://pypi.org/project/sentry-sdk/

Reply:
https://github.com/OCA/server-tools/issues/2927
looks like on 18.0 can use 3.12 to bypass
https://github.com/odoo/odoo/blob/18.0/requirements.txt#L80

https://github.com/OCA/server-tools/pull/3174/commits/a1969c2d2cc33221882a07f56b1ecfecc3ae531f
started the testing